fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -964,6 +968,22 @@ cleanTabletSchedQClause
     : CLEAN TABLET SCHEDULER QUEUE
     ;
 
+decommissionDiskClause
+    : DECOMMISSION DISK string (',' string)*
+    ;
+
+cancelDecommissionDiskClause
+    : CANCEL DECOMMISSION DISK string (',' string)*
+    ;
+
+disableDiskClause
+    : DISABLE DISK string (',' string)*
+    ;
+
+cancelDisableDiskClause
```

Comment:
we should also have a syntax sugar like `show proc "/backends/*";`, otherwist it's quite cubersome to should the disk state of all backends.